ckan_cloud_operator/deis_ckan/envvars.py

```diff
@@ -57,10 +57,10 @@ def _update(self):
             postgres_host, postgres_port = db_manager.get_internal_proxy_host_port()
             logs.info(f'Connecting to DB proxy: {postgres_host}:{postgres_port}')
         envvars.update(
-            CKAN_SQLALCHEMY_URL=f"postgresql://{db_name}:{db_password}@{postgres_host}:{postgres_port}/{db_name}",
-            CKAN___BEAKER__SESSION__URL=f"postgresql://{db_name}:{db_password}@{postgres_host}:{postgres_port}/{db_name}",
-            CKAN__DATASTORE__READ_URL=f"postgresql://{datastore_ro_user}:{datastore_ro_password}@{postgres_host}:{postgres_port}/{datastore_name}",
-            CKAN__DATASTORE__WRITE_URL=f"postgresql://{datastore_name}:{datastore_password}@{postgres_host}:{postgres_port}/{datastore_name}",
+            CKAN_SQLALCHEMY_URL=f"postgresql://{db_name}@{postgres_host}:{db_password}@{postgres_host}:{postgres_port}/{db_name}",
+            CKAN___BEAKER__SESSION__URL=f"postgresql://{db_name}@{postgres_host}:{db_password}@{postgres_host}:{postgres_port}/{db_name}",
+            CKAN__DATASTORE__READ_URL=f"postgresql://{datastore_ro_user}@{postgres_host}:{datastore_ro_password}@{postgres_host}:{postgres_port}/{datastore_name}",
+            CKAN__DATASTORE__WRITE_URL=f"postgresql://{datastore_name}@{postgres_host}:{datastore_password}@{postgres_host}:{postgres_port}/{datastore_name}",
```

Comment:
Could you please explain this? Why do we need `postgres_host` before username as well?

Reply:
Also, I remember I had a problem with connecting to azure database with URIs like this as username include `@` in it. Eg `psql postgresql://irakli.mchedlishvli@azure.com:passowrd@host:5000` was erroring cause it's getting confused with two `@`s in URL. Have we tested CKAN is fine with that?